setup.py

```diff
@@ -34,6 +34,12 @@
 with open(os.path.join(here, "README.rst")) as f:
     long_description = f.read()
 
+if sys.platform == 'win32':
+    kwds = {'data_files': [("cmd", ["cmd/activate", "cmd/deactivate",
+                                    "cmd/activate.bat", "cmd/deactivate.bat"]), ]
+            }
+else:
+    kwds = {'data_files':  [("cmd", ["cmd/activate", "cmd/deactivate"]), ]}
```

Comment:
@ilanschnell doesn't like the `cmd` prefix.  I agree with you that they're not quite bin files either.  Still, I'd be more comfortable with something that's more FHS-esque.